com.unity.multiplayer.mlapi/Editor/NetworkManagerEditor.cs

```diff
@@ -167,6 +167,24 @@ private void OnEnable()
         m_NetworkPrefabsList = new ReorderableList(serializedObject, serializedObject.FindProperty(nameof(NetworkManager.NetworkConfig)).FindPropertyRelative(nameof(NetworkConfig.NetworkPrefabs)), true, true, true, true);
         m_NetworkPrefabsList.drawElementCallback = (Rect rect, int index, bool isActive, bool isFocused) =>
         {
+            for (int i = 0; i < m_NetworkManager.NetworkConfig.NetworkPrefabs.Count; i++)
+            {
+                // Find the first playerPrefab
+                if (m_NetworkManager.NetworkConfig.NetworkPrefabs[i].PlayerPrefab)
+                {
+                    // Iterate over all other and set player prefab to false
+                    for (int j = 0; j < m_NetworkManager.NetworkConfig.NetworkPrefabs.Count; j++)
+                    {
+                        if (j != i && m_NetworkManager.NetworkConfig.NetworkPrefabs[j].PlayerPrefab)
+                        {
+                            m_NetworkManager.NetworkConfig.NetworkPrefabs[j].PlayerPrefab = false;
```

Comment:
also, I wonder if there should be a runtime check similar to this where we assert/log if there are more than one network prefabs marked as player prefab. what do you think?

Reply:
There is already. This only fixes the bug where you cannot disable the PlayerPrefab checkbox on newly created entries.